### PR TITLE
Update teams-store-validation-guidelines.md

### PR DESCRIPTION
**Pull request by ramendhe**
Added update in Tab navigation.

Please also change heading for this image to 'Left navigation in the tab conflicts with Primary Teams Validation'
image

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64cb428af60d500f8fb2197b4e880a7f024924a4/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 94 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/31fa3fd2-fc67-4c45-b07c-280b95748d90) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md [hunk 1]
```diff
@@ -730,7 +730,7 @@ Design your app with [basic](~/concepts/design/design-teams-app-basic-ui-compone
 
 :::image type="content" source="../../../../assets/images/submission/validation-navigation-left-navigation.png" alt-text="Graphic shows an example of left navigation rail that conflicts with the primary Teams navigation.":::
 
-* Tabs with toolbar in left rail must leave 20 pixels spacing from Teams left navigation. [*Mandatory Fix*]
+* If your tab has a toolbar on left side without any navigational components, toolbar must leave 20 pixels spacing from Teams left navigation. [*Mandatory Fix*]
```
Comment (v-ypalikila) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md:733]:
```suggested
If your tab has a toolbar on the left rail without any navigation component, the toolbar must leave 20 pixels spacing from Teams left navigation. [*Mandatory Fix*]
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4827bb7df1a632207d26276c402fd5d90b422246/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 94 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/384e96d8-f4f8-4de4-b995-7f630a889a63) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/10f2f29fdd12f6d86908cf330026759b7c0665e5/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 94 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/942e1638-d7ed-4db0-a752-7c5f1f6ff70d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/37113d2563073e404ed62ad55967b2601e857033/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 94 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4dd07d4-ed43-4367-af4c-e621000e1dd0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-ypalikila** [comment]
As the PR is having issues with Build. Raised a new [PR 7683](https://github.com/MicrosoftDocs/msteams-docs/pull/7683) with the same changes and merged the PR.

**Pull request closed without merging**